shared/src/commonMain/kotlin/com/example/tasky/model/agenda/Event.kt

```diff
@@ -1,8 +1,56 @@
 package com.example.tasky.model.agenda
 
 import io.ktor.resources.Resource
+import kotlinx.serialization.Serializable
 
 @Resource("/event")
 class EventPath
 
-typealias UpdateEventBody = Event
+@Serializable
+data class UpdateEventBody(
+    val id: String,
+    val title: String,
+    val description: String,
+    val from: Long,
+    val to: Long,
+    val remindAt: Long,
+    val attendeeIds: List<String>,
+    val deletedPhotoKeys: List<String>,
+    val isGoing: Boolean,
+    val photos: List<ByteArray>,
+) {
+    constructor(event: Event, deletedPhotoKeys: List<String>, isGoing: Boolean, photos: List<ByteArray>) : this(
```

Comment:
These classes look like they belong in the data layer 🤔
The reason is that the API endpoints require the model of the body; thus, this does not belong in the domain layer.